lib/checkPullRequestLabels.js

```diff
@@ -0,0 +1,40 @@
+module.exports.checkPullRequestLabels = async function(context) {
+  var pullRequest = context.payload.pull_request;
+  var pullRequestNumber = pullRequest.number;
+
+  // eslint-disable-next-line no-console
+  console.log('RUNNING LABEL CHECKS ON PULL REQUEST ' + pullRequestNumber + ' ...');
+
+  var labels = pullRequest.labels;
+  var userName = pullRequest.user.login;
+  var projectOwnerLabel;
+  var hasProjectOwnerLabel = labels.some(function(label) {
+    projectOwnerLabel = label.name;
+    return projectOwnerLabel.startsWith('PROJECT');
+  });
+
+  // If the PR has a project owner label, assign the project owner
+  // for the first-pass review of the PR and leave a top-level comment.
+  if(hasProjectOwnerLabel === true) {
+    var labelSubstrings = projectOwnerLabel.split('@');
```

Comment:
Make sure to deal correctly with the case where the project owner labels are not formatted correctly (multiple '@', no '@", project owner not found in list of People for oppia, etc.)

Reply:
The label should always be in this particular format: `PROJECT: <Project-Name> -- @<Owner-Name>` as defined in the [PR Review Cycle doc](https://docs.google.com/document/d/1y2_3Ns0V00QESZy3gswc413Rki-m1exTNDt3uVcu7w0/edit#).

Reply:
That is the intended format, but mistakes happen. The code should be robust to mistakes and not assume that the world around it is perfectly ordered.

Reply:
Sorry, but we will be modifying all the project labels according to this format for the bot to retrieve the project owner's names. There is no other way to get the name of the project owner, therefore, I am sure how to make this more robust.

Reply:
No, what I'm trying to say is -- let's say there's a typo in the project owner. Or there's an extra space after the "--". Or someone just goes ahead and adds a label. Or a project owner leaves the codebase.

I'm not saying you need to somehow handle all these cases one by one, but what I am suggesting is that if the "happy path" doesn't work you have some sort of fallback. Basically, what should happen if an error occurs with this automatic process?

Reply:
Okay, for extra spaces in the names, the process will work as-is since we're trimming the spaces. For any of the other scenarios, the process will never crash -- it will work but if suppose there's a typo, the project owner name will be incorrect and this will be reflected in the comment, and can, therefore, be corrected by just modifying the label. Or if the project owner leaves, the process will still work and can again be rectified by the comment left by the bot.
Basically, the comment created by the bot will always tell us what's going wrong with the labels.

Reply:
Let's say someone else adds a new project owner label that has no '@' at all, and then adds that label to their PR. What happens then?

Basically, I am trying to suggest the idea of a try/except to catch unforeseen cases.

Reply:
For this case, no one will be assigned to the PR and comment will be created which has no project owner. This will make it clear that the label is incorrectly formatted. I can add a try/except for this case where the project owner is an empty string if you'd like. Thanks!

Reply:
Here's what I suggest:

- On obtaining the label, validate that it has exactly the correct format. If it does not, then have the comment explicitly alert the dev workflow team that a project owner cannot be assigned and that there may be a problem with Oppiabot, so that they can look into it.
- After validating the label has the correct format, you can parse it in the usual way, secure in the assurance that the format is good and that there will be no surprises.

Sounds good?

Reply:
I have added a regexp test for validation. PTAL, thanks!